test/extended/networking/util.go

```diff
@@ -156,6 +159,16 @@ func pluginIsolatesNamespaces() bool {
 	return os.Getenv("OPENSHIFT_NETWORK_ISOLATION") == "true"
 }
 
+func makeGlobalNamespace(ns *api.Namespace) {
```

Comment:
(No action required) makeNamespaceGlobal since it's modifying an existing namespace to be global?